### ISSUE TITLE
Cannot find module 'winston/lib/winston'

### ISSUE THREAD

**Issue by dominicbarnes** [BohdanTkachenko/eslint-plugin-require-path-exists#14]
After upgrading to v1.1.4, I am seeing this error with the [winston](https://www.npmjs.com/package/winston) module.

```
   4:15  error    Cannot find module 'winston/lib/winston'       require-path-exists/exists
```

I don't recall exactly which version I had before, but this did not throw an error before. (I can go backwards to figure out which version broke if you'd like)

Fwiw, there is some ambiguity in the winston package, as their `main` is "./lib/winston", along with both `lib/winston/` and `lib/winston.js` existing in the repository. (however, that just means there are 2 files that match so this ESLint rule should not fail regardless)

**Comment by dominicbarnes** [comment]
I've discovered that the problem shows up between `1.0.25` and `1.1.0`. (ie: it works in the former, and breaks in the latter)

So, something in `1.1.0` breaks this particular edge-case.

**Comment by BohdanTkachenko** [comment]
I saw similar issue, but with another NPM package. But I thought that I really don't have this package and just did `npm install` for this package. As a quick fix please try to do the same.

And I will try to reproduce it again and to fix it.

**Comment by dominicbarnes** [comment]
I double-checked, the package is definitely installed.

Besides, the problem goes away when I downgrade to `1.0.25`, so it definitely is not an issue of the package being missing.

**Comment by BohdanTkachenko** [comment]
I mean that package is installed, but installing it again may help (don't know why yet). At least it helped me when I had this issue.

**Comment by dominicbarnes** [comment]
Oh, I understand now. I'll try that real quick.

**Comment by dominicbarnes** [comment]
I tried removing and reinstalling `winston`, that did not fix it.

**Comment by BohdanTkachenko** [comment]
Ok, I will to reproduce it.

**Comment by dominicbarnes** [comment]
Btw, is there any reason you aren't using something like [resolve](https://www.npmjs.com/package/resolve) to handle figuring out which file is being `require`'d? (rather than writing up that logic yourself) I think you can reduce the complexity/scope of this plugin in the process.

**Comment by BohdanTkachenko** [comment]
There is no reason, I just didn't know about this package. Thanks for the link, I will look at it to see if it can be used here.

**Comment by BohdanTkachenko** [comment]
Fixed in 1.1.5 (https://github.com/BohdanTkachenko/eslint-plugin-require-path-exists/commit/694c831830130353ee8a5e205aedbe93c9fb512b)

Great thanks @dominicbarnes for idea of using resolve package. It reduced code a lot.

Feel free to reopen this issue if bug is not fixed.

**Comment by dominicbarnes** [comment]
Awesome! I look forward to upgrading once the next release is published :)

**Comment by BohdanTkachenko** [comment]
It's already published, so you can use it :)

**Comment by dominicbarnes** [comment]
It works! Thanks again :)
